src/test/java/us/kbase/test/auth2/lib/AuthenticationTestModeTokenTest.java

```diff
@@ -71,7 +72,7 @@ public void createTokenWithoutName() throws Exception {
 		when(clock.instant()).thenReturn(Instant.ofEpochMilli(10000));
 		when(rand.getToken()).thenReturn("whee");
 		
-		final NewToken nt = auth.testModeCreateToken(new UserName("foo"), null, TokenType.AGENT);
```

Comment:
Need to add tests for other mfa types. This could be done by just modifying existing tests I think